src/objects/Mesh.cpp

```diff
@@ -119,12 +110,22 @@ void Mesh::initVerticesFromAiMesh(const aiMesh *paiMesh, Chunk &chunk) {
         chunk.m_normals.push_back(make_vector(pNormal.x, pNormal.y, pNormal.z));
         chunk.m_uvs.push_back(make_vector(pTexCoord.x, pTexCoord.y));
 
-        if(paiMesh->HasVertexColors(0)) {
-            aiColor4D vertexColor = paiMesh->mColors[0][i];
-            chunk.m_vertexColors.push_back(chag::make_vector(vertexColor.r, vertexColor.g, vertexColor.b));
-        } else {
-            chunk.m_vertexColors.push_back(chag::make_vector(0.0f, 0.0f, 0.0f));
-        }
+        unsigned int j = 0;
```

Comment:
Previously, we only loaded the first vertex color buffer, but there can be many!